docker-compose.yaml

```diff
@@ -13,8 +13,9 @@ services:
     networks:
       - discount-bandit
     volumes:
-      - ./database/database.sqlite:/app/database/database.sqlite
+      - ./database/:/app/database/
```

Comment:
i don't want the migration files to be shared to the user, what you can do is edit the code to put database.sqlite inside a folder like /app/database/data/ and then link that. then i will approve this part as i agree this is easier

Reply:
Ok, I can do that as well.

That's a breaking change which will require users to manually move the database though.
Unless I missed something?

Reply:
i can move the database using a script, but the user needs to add the new volume.

but i'll test it and see what we can do to make it easier to migrate for user ( maybe adding a soft link will not require any change from user side)